airflow/data/agencies.yml

```diff
@@ -1394,17 +1394,17 @@ ventura-county-transportation-commission:
   agency_name: Ventura County Transportation Commission
   feeds:
     - gtfs_schedule_url: http://data.trilliumtransit.com/gtfs/vctc-ca-us/vctc-ca-us.zip
-      gtfs_rt_vehicle_positions_url: null
-      gtfs_rt_service_alerts_url: null
-      gtfs_rt_trip_updates_url: null
+      gtfs_rt_vehicle_positions_url: https://govcbus.com/gtfs-rt/vehiclepositions
+      gtfs_rt_service_alerts_url: https://govcbus.com/gtfs-rt/alerts
+      gtfs_rt_trip_updates_url: https://govcbus.com/gtfs-rt/tripupdates
```

Comment:
AFAIK, here might be worthwhile to see if we should use theGMV vs Trillium feeds (and / or diff em)

Reply:
Was verified by Evan that the VCTC RT feeds were marked as the Trillium on the transit database